pkg/controller/migration/namespace_migration.go

```diff
@@ -746,3 +791,16 @@ func (m *CoreNamespaceMigration) removeNodeLabel(ctx context.Context, nodeName,
 		return true, nil
 	})
 }
+
+// remove temporary ClusterRole and CLusterRoleBinding that gives calico-node permission to create servicetoken in kube-system
+func (m *CoreNamespaceMigration) deleteTemporaryCalicoNodeClusterRole(ctx context.Context) error {
+	err := m.client.RbacV1().ClusterRoleBindings().Delete(ctx, calicoNodeMigrationName, metav1.DeleteOptions{})
+	if err != nil {
+		return err
```

Comment:
Instead of exiting here, can we collect the errors in a slice and return the errors at the end of this function? As is, if something prevents the role binding from being deleted, we will not delete the cluster role.